.github/workflows/stale.yml

```diff
@@ -0,0 +1,27 @@
+name: Stale Bot
+
+on:
+  schedule:
+    - cron: "0 15 * * *"
+
+jobs:
+  close_stale_issues:
+    name: Close Stale Issues
+    if: github.repository == 'huggingface/peft'
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think this secret should be present on the repository , not sure if that's the case

Reply:
This is not needed to be added on the repository, check: https://docs.github.com/en/actions/security-guides/automatic-token-authentication